test/dash/dash_parser_content_protection_unit.js

```diff
@@ -741,3 +741,113 @@ describe('DashParser ContentProtection', function() {
     await Dash.testFails(source, expected);
   });
 });
+
+describe('In-manifest PlayReady and Widevine', function() {
+  const ContentProtection = shaka.dash.ContentProtection;
+  const strToXml = function(str) {
+    const parser = new DOMParser();
+    return parser.parseFromString(str, 'application/xml').documentElement;
+  };
+
+  describe('getWidevineLicenseUrl', function() {
+    it('valid ms:laurl node', function() {
+      const input = {
+        init: null,
+        keyId: null,
+        schemeUri: '',
+        node: strToXml([
+          '<test xmlns:ms="urn:microsoft">',
```

Comment:
I don't think this isn't the correct namespace URI.

Reply:
Couldn't find much on this but I did find this [Azure Media sample](https://amp.azure.net/libs/amp/latest/docs/samples.html) page that links to this [manifest](https://ampvideos.streaming.mediaservices.windows.net/9b77ed82-94af-41bc-b08b-537e7a530eb6/sintel_trailer-1080p_fp.ism/manifest(format=mpd-time-csf)) which uses the same namespaces so I think it's correct:
```<?xml version="1.0" encoding="utf-8"?>
<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" 
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    profiles="urn:mpeg:dash:profile:isoff-live:2011" type="static" 
    xmlns:cenc="urn:mpeg:cenc:2013" 
    xmlns:mspr="urn:microsoft:playready" 
    xmlns:ms="urn:microsoft" mediaPresentationDuration="PT52.208S" minBufferTime="PT3S">
    <Period>
        <AdaptationSet ...snip...>
            <ContentProtection schemeIdUri="urn:mpeg:dash:mp4protection:2011" value="cenc" cenc:default_KID="096D74B9-362D-4110-949B-EC0F5694F750"/>
            <ContentProtection schemeIdUri="urn:uuid:9A04F079-9840-4286-AB92-E65BE0885F95" value="2.0" cenc:default_KID="096D74B9-362D-4110-949B-EC0F5694F750">
                <mspr:pro>...snip...</mspr:pro>
            </ContentProtection>
            <ContentProtection schemeIdUri="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed" cenc:default_KID="096D74B9-362D-4110-949B-EC0F5694F750">
                <cenc:pssh>AAAAMnBzc2gAAAAA7e+LqXnWSs6jyCfc1R0h7QAAABISEAltdLk2LUEQlJvsD1aU91A=</cenc:pssh>
                <ms:laurl licenseUrl="https://ampvideos.keydelivery.mediaservices.windows.net/Widevine/?kid=096d74b9-362d-4110-949b-ec0f5694f750"/>
            </ContentProtection>
...snip...
```